marathon_acme/tests/test_clients.py

```diff
@@ -532,7 +605,7 @@ def test_get_json_field(self):
 
         request = yield self.requests.get()
         self.assertThat(request, HasRequestProperties(
-            method='GET', url=self.uri('/my-path')))
+            method='GET', url='http://localhost:8080/my-path'))
```

Comment:
Can we make `self.uri()` work for the common case instead of having to explicitly provide the full URI everywhere?